src/functions/helper_matrices.ts

```diff
@@ -30,6 +33,10 @@ export function invertMatrix(M: Matrix<number>): {
   // (b) Multiply a row by a scalar. This multiply the determinant by that scalar.
   // (c) Add to a row a multiple of another row. This does not change the determinant.
 
+  if (M.length < 1 || M[0].length < 1) {
+    throw new Error(_t("Function [[FUNCTION_NAME]] invert matrix error, empty matrix"));
```

Comment:
user-facing errors should be `EvaluationError`
```suggestion
    throw new EvaluationError(_t("Function [[FUNCTION_NAME]] invert matrix error, empty matrix"));
```